roles/efk_opensearch_stack/tasks/install_fluentd.yaml

```diff
@@ -0,0 +1,16 @@
+---
+- name: "Install Fluentd"
+  kubernetes.core.helm:
+    chart_repo_url: "{{ elastic_fluentd_chart_repo }}"
+    chart_ref: "{{ elastic_fluentd_chart_name }}"
+    chart_version: "{{ elastic_fluentd_chart_version }}"
+    release_namespace: "{{ elastic_opensearch_namespace }}"
+    release_name: "fluentd"
+    release_state: present
+    release_values: "{{ elastic_fluentd_values }}"
+
+- name: "Wait for Fluentd rollout"
```

Comment:
nit: I got the feeling that `shell: ...` should be in second line here. That seems like a more common case from I've seen.